network/ocsp.go

```diff
@@ -85,54 +85,51 @@ func NewOCSPConfig(uri, required, fromCert string) (*OCSPConfig, error) {
 		// TODO: Do some request to `uri`, log warn if failed
 	}
 
-	var requiredVal int
-	switch required {
-	case ocspRequiredYesStr:
-		requiredVal = ocspRequiredYes
-	case ocspRequiredNoStr:
-		requiredVal = ocspRequiredNo
-	case ocspRequiredAllStr:
-		requiredVal = ocspRequiredAll
-	default:
-		return nil, errors.New("Invalid `ocsp_required` value '" + required + "', should be one of 'yes', 'no', 'all'")
+	requiredValValues := map[string]int{
+		ocspRequiredYesStr: ocspRequiredYes,
+		ocspRequiredNoStr:  ocspRequiredNo,
+		ocspRequiredAllStr: ocspRequiredNo,
 	}
```

Comment:
I wonder if there is any merit in having parallel lists of `ocspRequired${foo}Str` and `ocspRequired${foo}` instead of just using a single set of named string constants everywhere...